youtube_dl/extractor/mediahuis.py

```diff
@@ -0,0 +1,189 @@
+                               'steenweg in het Oost-Vlaamse Erpe-Mere. De BMW kwam van de oprit E40 gere...',
+                'thumbnail': 're:http.*jpg$',
+                'upload_date': '20170413',
+                'uploader_id': 'video',
+                'timestamp': 1492092669,
+            }
+        },
+        # De Standaard (Kaltura, Dynamic Embed)
+        {
+            'url': 'http://www.standaard.be/cnt/dmf20170412_02831280',
+            'md5': '490f0874ad308a6010292edd1a041b14',
+            'info_dict': {
+                'id': '02831280',
+                'ext': 'mp4',
+                'title': 'Zo klinkt een gitaar gemaakt van papier',
+                'description': 'Gitaarbouwer Walter Verreydt bouwde een gitaar uit krantenpapier. Kan een papieren '
+                               'gitaar zo goed klinken als een gitaar van tropisch hout? Blijkbaar wel. ...',
+                'thumbnail': 're:http.*jpg$',
+                'upload_date': '20170412',
+                'uploader_id': 'videoredactie@mediahuis.be',
+                'timestamp': 1492009206,
+            }
+        },
+        # De Standaard (Kaltura, Auto Embed)
+        {
+            'url': 'http://www.standaard.be/cnt/dmf20170413_02832378',
+            'md5': 'c3be760e20a9b49b3eac2536bc045b5b',
+            'info_dict': {
+                'id': '02832378',
+                'ext': 'mp4',
+                'title': 'VIDEO. Is bio ons geld waard?',
+                'description': 'Is biologisch eten beter voor de wereld, of een sprookje voor naïeve consumenten? '
+                               'Correspondente en voedingsjournaliste Dorien Knockaert neemt de kritiek v...',
+                'thumbnail': 're:http.*jpg$',
+                'upload_date': '20170412',
+                'uploader_id': 'batchUser',
+                'timestamp': 1491990547,
+            }
+        }
+    ]
+
+    def _real_extract(self, url):
+        """ Extract the video info from the given mediahuis URL """
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        thumbnail = self._og_search_thumbnail(webpage)
+        title = self._og_search_title(webpage)
+        description = self._og_search_description(webpage)
+        url_type = None
+
+        iframe_m = re.search(r'<script[^>]+src="(.+?kaltura.com.*?)"', webpage)
+        if iframe_m:
+            video_url = smuggle_url(KalturaIE._extract_url(webpage), {'source_url': url})
+            url_type = 'url_transparent'
+
+        iframe_m = re.search(r'<iframe[^>]+src="(.+?vier.be.*?)"', webpage)
+        if iframe_m:
+            video_url = (iframe_m.group(1))
+            url_type = 'url_transparent'
+
+        iframe_m = re.search(r'<iframe[^>]+src="(.+?vrt.be.*?)"', webpage)
+        if iframe_m:
+            webpage = self._download_webpage(iframe_m.group(1), "vrt-iframe")
+            video_url = self._search_regex(r'sources.pdl = "(.*?)";', webpage, 'vrt-video')
```

Comment:
1. Dots should be escaped
2. Better to use ```\s*``` or ```\s+``` instead of spaces in regular expressions for robustness